pylint/utils/file_state.py

```diff
@@ -162,7 +166,10 @@ def iter_spurious_suppression_messages(
                 if not enable and (warning, line) not in self._ignored_msgs:
                     # ignore cyclic-import check which can show false positives
                     # here due to incomplete context
-                    if warning != "R0401":
+                    # and deprecated-{module, argument, class, method, decorator}
+                    # which can cause false positives for multi-interpreter projects
+                    # when linting with an interpreter on a lower python version
```

Comment:
Can you move the explanation to the constant file? I think it would make more sense there.